packages/components/src/file-upload/FileUpload.tsx

```diff
@@ -36,6 +38,7 @@ export const FileUpload: React.FC<FileTriggerProps> = ({
   ...rest
 }) => {
   const [files, setFiles] = React.useState<FileState>(null)
+  const strings = useLocalizedStringFormatter(messages)
 
   const handleUpload = (files: FileList | null) => {
```

Comment:
```suggestion
  const handleUpload = (files: FileList | null) => {
    if (rest.onSelect) {
      rest.onSelect(files)
    }
```

Ska man göra något sånthär? Alternativt plocka bort möjligheten att använda onSelect